src/platform.ts

```diff
@@ -437,14 +455,16 @@ export class PlatformInformation {
             });
     }
 
-    private static getWindowsArchitectureEnv(): Promise<string> {
-        return new Promise<string>((resolve, reject) => {
-            if (process.env.PROCESSOR_ARCHITECTURE === 'x86' && process.env.PROCESSOR_ARCHITEW6432 === undefined) {
-                resolve('x86');
-            } else {
-                resolve('x86_64');
-            }
-        });
+    private static async getWindowsArchitectureEnv(): Promise<string> {

```

Comment:
Actually - I suppose you could just make this return the string directly as well. No reason for it to even be async

Reply:
lol, not sure why it was like that in the first place.